### DISCUSSION TITLE
Update not availible

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1233] [Fixed Issues]
The new Update seems to be not availible.
After new install the System wants to downgrade to 20230205.

**Comment by emesimon** [comment]
MPD is not running --
 MPD is not running 

mpd 0.23.12-1
# /srv/http/data/mpdconf/mpd.conf
audio_buffer_size       "4096"
auto_update             "yes"
bind_to_address         "0.0.0.0"
db_file                 "/srv/http/data/mpd/mpd.db"
max_output_buffer_size  "8192"
music_directory         "/mnt/MPD"
playlist_directory      "/srv/http/data/playlists"
state_file              "/var/lib/mpd/mpdstate"
user                    "mpd"
input {
	plugin  "curl"
}
audio_output {
	name           "HiFiBerry DAC+"
	device         "hw:0,0"
	type           "alsa"
	auto_resample  "no"
	auto_format    "no"
	mixer_type     "software"
	dop            "yes"
}

mpd.service- Music Player Daemon
     Loaded: loaded (/usr/lib/systemd/system/mpd.service; disabled; preset: disabled)
    Drop-In: /usr/lib/systemd/system/mpd.service.d
             └─00-arch.conf
             /etc/systemd/system/mpd.service.d
             └─override.conf
     Active: failed (Result: timeout) since Sun 2023-02-12 15:08:47 UTC; 13min ago
   Duration: 36.654s
       Docs: man:mpd(1)
             man:mpd.conf(5)
    Process: 5667 ExecStart=/usr/bin/mpd --systemd /srv/http/data/mpdconf/mpd.conf (code=exited, status=0/SUCCESS)
   Main PID: 5667 (code=exited, status=0/SUCCESS)
        CPU: 16.000s
Feb 12 15:08:08 rAudio mpd[5667]: inotify: Failed to register /mnt/MPD/USB/Tos2TB/00_neu/BOB DYLAN & THE GRATEFUL DEAD 1987: inotify_add_watch() has failed: Permission denied
Feb 12 15:08:08 rAudio mpd[5667]: inotify: Failed to register /mnt/MPD/USB/Tos2TB/00_neu/NEW ORDER - SAN DIEGO 1987: inotify_add_watch() has failed: Permission denied
Feb 12 15:08:09 rAudio mpd[5667]: inotify: Failed to register /mnt/MPD/USB/Tos2TB/00_neu/DOLORES (CRANBERRIES) O’RIORDAN R.I.P. 1971 - 2018: inotify_add_watch() has failed: Permission denied
Feb 12 15:08:10 rAudio mpd[5667]: inotify: Failed to register /mnt/MPD/USB/Tos2TB/00_neu/JAZZ ON SUNDAY_ PETER BROTZMANN, KEIJI HAINO - VIENNA 2018: inotify_add_watch() has failed: Permission denied
Feb 12 15:08:10 rAudio mpd[5667]: inotify: Failed to register /mnt/MPD/USB/Tos2TB/00_neu/VARIOUS - LEONARD COHEN TRIBUTE, MONTREAL 2017: inotify_add_watch() has failed: Permission denied
Feb 12 15:08:25 rAudio mpd[5667]: inotify: Failed to register /mnt/MPD/USB/Tos2TB/lost+found: inotify_add_watch() has failed: Permission denied
Feb 12 15:08:46 rAudio systemd[1]: mpd.service: start operation timed out. Terminating.
Feb 12 15:08:47 rAudio systemd[1]: mpd.service: Failed with result 'timeout'.
Feb 12 15:08:47 rAudio systemd[1]: Failed to start Music Player Daemon.
Feb 12 15:08:47 rAudio systemd[1]: mpd.service: Consumed 16.000s CPU time.

**Comment by ceejayemm** [comment]
@emesimon - I think your problem should be reported as a separate issue as its probably a different problem. BTW I DON'T have your problem with the RPi 2 (32 bit) i20230212 image.

@hjfisch - I CAN confirm the same issue as you in regard to trying to apply updates. As noted above I installed the RPi 2 (32 bit) i20230212 image but the System page reports the build as:

Version: rAudio 20230205     (<-- note NOT 20230212)
Kernel:   6.1.11-1 armv71       (but indicates the new kernel version)

The  Settings Cog is Blue and the Updates pages reports 20230205 updates are available.  I haven't tried to apply the available updates at this point.

Chris

**Comment by Vyacheslav-S** [comment]
RPi3B+ (64bit) - the update (20230212) appeared about an hour ago, it was not displayed earlier.
I updated and everything went well.
